tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "strictNullChecks": true,
     "esModuleInterop": true,
     "noImplicitAny": true,
-    "module": "es6",
```

Comment:
Had to remove this because ts-node was compiling ts to es6 modules instead of commonjs and mocha running on node couldn't handle them.